### DISCUSSION TITLE
'fastdeploy' has no attribute 'download_and_decompress'

### DISCUSSION THREAD

**Discussion by mk12306** [PaddlePaddle/FastDeploy#551] [Q&A]
How to solve this problem? 
fastdeploy-python==0.6.0

Traceback (most recent call last):
  File "fastdeploy.py", line 2, in <module>
    import fastdeploy as fd
  File "/root/workspace/FastDeploy/fastdeploy.py", line 8, in <module>
    fd.download_and_decompress(model_url, path=".")
AttributeError: partially initialized module 'fastdeploy' has no attribute 'download_and_decompress' (most likely due to a circular import)

**Comment by jiangjiajun** [comment]
There's name confilct, please modify your code file name from "fastdeploy.py" to "test.py" and try again

**Reply by mk12306** [reply]
> There's name confilct, please modify your code file name from "fastdeploy.py" to "test.py" and try again

ok，thx

**Comment by mk12306** [comment]
new problem:
opencv-python-4.6.0.66

objc[59243]: Class CVView is implemented in both /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/cv2/cv2.abi3.so (0x128a0fb00) and /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/fastdeploy/libs/third_libs/opencv/lib/libopencv_highgui.3.4.dylib (0x14b1b7470). One of the two will be used. Which one is undefined.

**Reply by jiangjiajun** [reply]
Is this a warning or error?

**Reply by mk12306** [reply]
The program was interrupted directly

Process finished with exit code 132 (interrupted by signal 4: SIGILL)

**Reply by jiangjiajun** [reply]
could you describle your environment information

OS version
CPU Arch
Python version

**Reply by mk12306** [reply]
this is my environment information:
macOS Big Sur
Apple M1
Python version: 3.8

**Reply by jiangjiajun** [reply]
Hi, We haven't release fastdeploy-python 0.6.0 on MacOS arm version yet, so is this fastdeploy-python is built by your self?

Could you install a Anaconda(Mac arm64 version)， and install this wheel try again?
```
pip install https://bj.bcebos.com/fastdeploy/release/wheels/fastdeploy_python-0.6.0-cp39-cp39-macosx_11_0_arm64.whl
```

**Reply by jiangjiajun** [reply]
This is my test result
![image](https://user-images.githubusercontent.com/19339784/201462476-554ffc39-5d57-466d-ba0d-7d1f2e4453dc.png)
